mappings/net/minecraft/command/argument/BlockArgumentParser.mapping

```diff
@@ -5,12 +5,15 @@ CLASS net/minecraft/unmapped/C_byxnvdet net/minecraft/command/argument/BlockArgu
 	FIELD f_cvoqhxbz DUPLICATE_PROPERTY_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
 	FIELD f_dnhrjtrg INVALID_BLOCK_ID_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD f_dxmzxkbl PROPERTIES_CLOSING C
+	FIELD f_fqqwxkcy blocks Lnet/minecraft/unmapped/C_vtbxyypo;
 	FIELD f_frcmfegl SUGGEST_DEFAULT Ljava/util/function/Function;
 	FIELD f_hucgnvir PROPERTIES_OPENING C
 	FIELD f_kgowavap data Lnet/minecraft/unmapped/C_hhlwcnih;
 	FIELD f_labuxzvk INVALID_PROPERTY_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic3CommandExceptionType;
+	FIELD f_nxipzgze allowNbt Z
 	FIELD f_octvggph EMPTY_PROPERTY_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
 	FIELD f_oggzvuey NBT_OPENING C
+	FIELD f_qcqiwnvh ERROR_UNKNOWN_TAG Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```

Comment:
```suggestion
	FIELD f_qcqiwnvh UNKNOWN_TAG_ERROR Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```

Reply:
I'd change it to `UNKNOWN_TAG_EXCEPTION`
```suggestion
	FIELD f_qcqiwnvh UNKNOWN_TAG_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```